Content.Server/Buckle/Components/BuckleComponent.cs

```diff
@@ -139,16 +139,10 @@ public void ReAttach(StrapComponent strap)
             }
 
             // Assign BuckleOffset first, before causing a MoveEvent to fire
-            if (strapTransform.WorldRotation.GetCardinalDir() == Direction.North)
-            {
-                BuckleOffset = (0, 0.15f);
-                ownTransform.WorldPosition = strapTransform.WorldPosition + BuckleOffset;
-            }
-            else
-            {
-                BuckleOffset = Vector2.Zero;
-                ownTransform.WorldPosition = strapTransform.WorldPosition;
-            }
+            BuckleOffset = strap.BuckleOffset;
+            //ownTransform.Coordinates = strapTransform.Coordinates.Offset(BuckleOffset);
+
+            ownTransform.Coordinates = ownTransform.Coordinates.WithPosition(BuckleOffset);
```

Comment:
This was updated on master to actually work with grid rotation so this will need updating.